src/packages/admin-ui-components/src/dropdown/dropdown.tsx

```diff
@@ -51,28 +61,52 @@ export const Dropdown = ({
 	const DropDownList = useMemo(
 		() => (
 			<>
-				{items.map((item) => (
-					<li key={item.id}>
-						{!item.href ? (
-							<span onClick={handleOnClickItem(item)}>{item.name}</span>
-						) : (
-							<a href={item.href} onClick={handleOnClickItem(item)}>
-								{item.name}
-							</a>
-						)}
-						{item.renderAfter?.()}
-					</li>
-				))}
+				{items.map((item) => {
```

Comment:
You could `{items.map((item) => (` then remove the return keyword on the next line.